src/bot.ts

```diff
@@ -678,9 +681,11 @@ class LemmyBot {
                     read: true
                   });
 
-                  console.log(
-                    `Marked private message ID ${messageView.private_message.id} from ${messageView.creator.id} as read`
-                  );
+                  if(this.#enableLogs){
+                    console.log(
+                      `Marked private message ID ${messageView.private_message.id} from ${messageView.creator.id} as read`
+                    );
+                  }
```

Comment:
Given how much the pattern `if (this.#enableLogs) { console.log(something) }` repeats, I think this should be extracted into a function (preferably a private class method).

Reply:
Makes sense. On it.